src/basic/missing_fs.h

```diff
@@ -1,50 +1,8 @@
 /* SPDX-License-Identifier: LGPL-2.1-or-later */
 #pragma once
 
-/* temporary undef definitions in bits/uio-ext.h, which is included by sys/uio.h */
```

Comment:
BTW, I have no idea why the conflict between sys/uio.h and linux/fs.h is fixed now...
Both seem to have RWF_HIPRI and friends without any guard...